chapters/meta-information.adoc

```diff
@@ -114,17 +114,15 @@ groups with clear organisational and legal boundaries:
 **Note**: sub-categorised as **internal**, **partner** and **public** this affects which subdomain deploy the API Proxy
 is deployed to in **Tyk Gateway**
 
-*component-internal*::
```

Comment:
in use?

https://github.com/linz/landonline-notices/blob/master/openapi/api_specs_batch.yaml#L24

Reply:
Yes, I will update these.

https://toitutewhenua.atlassian.net/browse/ENABLE-4624